src/webots/gui/WbGuiApplication.cpp

```diff
@@ -122,6 +122,75 @@ void WbGuiApplication::restart() {
 #endif
 }
 
+void WbGuiApplication::parseStreamArguments(const QString &serverArgument) {
```

Comment:
argument or arguments?
Should be consistent in the function name and argument name.

```suggestion
void WbGuiApplication::parseStreamArguments(const QString &serverArguments) {
```